tests/core/engine_adapter/test_bigquery.py

```diff
@@ -364,3 +364,18 @@ def test_merge(mocker: MockerFixture):
         "job_config",
         "table",
     ]
+
+
+def test_begin_end_session(mocker: MockerFixture):
```

Comment:
Does it also make sense to ensure that a series of engine adapter calls within a single session share a common session ID on their jobs?

Reply:
Updated the test